mews-operations/availabilityBlock.md

```diff
@@ -33,4 +33,178 @@
+| `name` | `string` | required | Name of the availability block. |
+| `dates` | [`Dates`](#dates) object | required| Dates of the availability block. |
+| `releaseStrategy` | [`Release strategy`](#release-strategy) object | required | Release strategy of the availability block. |
+| `originalRateCode` | `string` | required | Original rated code of the availability block. |
+| `spaceCategories` | [`Space category allocation`](#space-category-allocation) collection | required | Allocated categories of the availability block. |
+| `booker` | [`Customer`](./reservations.md#customer) object | required | The main booker. This does not mean that the person has arrived at the property. |
+| `company` | [`Company`](./reservations.md#company) object | optional | The company associated with the availability block. |
+| `notes` | `string` | optional | Notes related to the availability block. |
+
+
+#### Dates
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `start` | `string` | required | Start date in format `yyyy-mm-dd`, e.g. `2024-12-05`, this is the first day of the block and so can be booked for stay. |
+| `end` | `string` | required | End date in format `yyyy-mm-dd`, e.g. `2024-12-08`, this day is _not_ included in the block and so cannot be booked for stay (it is analogous to reservation departure). |
+
+
+#### Release strategy
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `type` | `string` | [`Release strategy type`](#release-strategy-type) required | release strategy for the availability block. |
+| `fixedDate` | `string` | required | release date in format `"yyyy-MM-dd"` \(e.g., `"2024-12-08"`\). |
+
+
+#### Space category allocation
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `spaceTypeCode` | `string` | required | Space type code of the block. |
+| `allocatedSpaces` | [`Space count`](#space-count) collection | required | The number of spaces **allocated** in the time interval. |
+| `rates` | [`Rates`](#rate) collection | optional | Prices for the space category. |
+
+
+### Space count
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `start` | `string` | required | Start date in format `"yyyy-MM-dd"` \(e.g. `"2024-12-05"`\). |
+| `end` | `string` | required | End date \(excluded\) in format `"yyyy-MM-dd"` \(e.g., `"2024-12-08"`\). |
+| `count` | `int` | required | Number of spaces (base of usage either occupied or allocated.) |
+
+
+### Rate
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `start` | `string` | required | Start date in format `"yyyy-MM-dd"` \(e.g. `"2024-12-05"`\). |
+| `end` | `string` | required | End date \(excluded\) in format `"yyyy-MM-dd"` \(e.g., `"2024-12-31"`\). |
+| `grossAmount` | `string` | required | Price with taxes included. | 
+
+
+#### Status
+
+* `Created`
+* `Updated`
+* `Canceled`
+
+#### Release strategy type
```

Comment:
Is there only one release strategy type, or will others be added in the future? The Connector API team use the convention of adding an elipsis (...) for an extensible string enumeration. Without this, arguably it's a breaking change if you want to add additional values in the future. For example:
* `Fixed`
* ...

Reply:
we will add some items to it. So I will add ... to it